docs/excel/custom-functions-volatile.md

```diff
@@ -19,13 +19,15 @@ If choosing to autogenerate your JSON file, declare a volatile function with the
 
 An example of a volatile custom function follows, which simulates rolling a six-sided dice.
 
+![A gif showing a custom function returning a random value to simulate rolling a six-sided dice](../images/SixSideDice.gif)
+
 ```JS
 /**
  * Simulates rolling a 6-sided dice.
  * @customfunction
  * @volatile
  */
-function roll6sided(): number {
```

Comment:
Why is this return type changing? The function still returns a number.

Reply:
Yes but the sample is in JS and not TS, sample won't run for me in Excel without this change